runtime/interpreter/value.go

```diff
@@ -11515,6 +11553,13 @@ func NewCompositeValue(
 	address common.Address,
 ) *CompositeValue {
 
+	if interpreter.tracingEnabled {
+		startTime := time.Now()
+		defer func() {
+			interpreter.reportCompositeValueConstructTrace(address.String(), qualifiedIdentifier, kind.String(), time.Since(startTime))
```

Comment:
Please chop down the argument list for 3+ arguments (here and in the other cases below):

```suggestion
			interpreter.reportCompositeValueConstructTrace(
				address.String(), 
				qualifiedIdentifier, 
				kind.String(), 
				time.Since(startTime),
			)
```

Reply:
updated all calls